ethcore/src/account_provider/mod.rs

```diff
@@ -24,7 +24,8 @@ use std::fmt;
 use std::collections::HashMap;
 use std::time::{Instant, Duration};
 use util::RwLock;
-use ethstore::{SimpleSecretStore, SecretStore, Error as SSError, EthStore, EthMultiStore, random_string};
+use ethstore::{SimpleSecretStore, SecretStore, Error as SSError, EthStore, EthMultiStore
```

Comment:
Updated this file just to make it work with new ethstore API. Its own API will be updated in next PR[s] (about RPCs).